CHANGELOG.rst

```diff
@@ -1,5 +1,45 @@
 .. _changelog:
 
+0.8.2
+-----
+
```

Comment:
mention the new UI option?

Reply:
You mean the whole revamped ui, right? I will add that under improvements.